### DISCUSSION TITLE
I want to remove kbar/search icon

### DISCUSSION THREAD

**Discussion by sowonpark-polsci** [alshedivat/al-folio#2703] [Q&A] [answered]
Hi! I want to remove search icon(kbar) from the top bar. Where do I find this? Much appreciated!

**Comment by george-gca** [comment, marked as answer]
Set here to false:

https://github.com/alshedivat/al-folio/blob/046545983f0864b62e883105955717cbf298561c/_config.yml#L52

**Reply by sowonpark-polsci** [reply]
Thank you!!
